build_idfboot.sh

```diff
@@ -75,8 +92,11 @@ build_idfboot() {
   cp "${build_dir}"/bootloader.bin "${output_dir}"/bootloader-"${target}".bin
 
   # Generate partition table binary file
+  # shellcheck disable=SC2086 # Intentionally split words from variables
```

Comment:
Are we going to keep this to silence shellcheck?

Reply:
IDF also follows the same approach on some scripts:
https://github.com/espressif/esp-idf/blob/master/export.sh#L19
https://github.com/espressif/esp-idf/blob/master/export.sh#L47

I think it is okay. It prevents an eventual attempt of removing the warning by applying an improper fix.